dd-java-agent/instrumentation/jetty-8/src/main/java/datadog/trace/instrumentation/jetty8/JettyHandlerAdvice.java

```diff
@@ -50,7 +50,6 @@ public static Scope startSpan(
     Tags.COMPONENT.set(span, "jetty-handler");
     Tags.HTTP_METHOD.set(span, req.getMethod());
     Tags.HTTP_URL.set(span, req.getRequestURL().toString());
-    span.setTag(DDTags.RESOURCE_NAME, resourceName);
```

Comment:
Removing this and setting a sane operation name default is necessary at the moment to prevent the ResourceNameDecorator from applying the instrumentation-set value over that of the URLAsResourceName in some cases, as their application order isn't guaranteed: https://github.com/signalfx/signalfx-java-tracing/blob/master/dd-trace-ot/src/main/java/datadog/opentracing/DDTracer.java#L81